mmdet/models/detectors/maskformer.py

```diff
@@ -0,0 +1,107 @@
+import warnings
+
+from ..builder import DETECTORS, build_backbone, build_head, build_neck
+from .single_stage import SingleStageDetector
+
+
+@DETECTORS.register_module()
+class MaskFormer(SingleStageDetector):
+    r"""Implementation of `Per-Pixel Classification is
+    NOT All You Need for Semantic Segmentation
+    <https://arxiv.org/pdf/2107.06278>`"""
```

Comment:
<https://arxiv.org/pdf/2107.06278>` -> <https://arxiv.org/pdf/2107.06278>`_

Reply:
add _ in the end ??

Reply:
yes